README.md

```diff
@@ -1,6 +1,6 @@
 # [ShardingSphere - Building a Criterion and Ecosystem Above Multi-Model Databases](https://shardingsphere.apache.org/)
 
-**Official Website:** [https://shardingsphere.apache.org/](https://shardingsphere.apache.org/)
+**Official Website:** [https://shardingsphere.apache.org/]()
```

Comment:
This is a markdown grammar. Your change will trigger a bad redirection.

Reply:
Noted, fixed